osu.Game/Screens/Edit/Editor.cs

```diff
@@ -719,6 +719,12 @@ private void dimBackground()
 
         public override bool OnExiting(ScreenExitEvent e)
         {
+            // Before exiting, trigger a focus loss.
+            //
+            // This is important to ensure that if the user is still editing a textbox, it will commit
+            // (and potentially block the exit procedure for save).
+            GetContainingInputManager().TriggerFocusContention(this);
```

Comment:
I'm having a hard time accepting this workaround all the way up here in `Editor`. Can we at least add a virtual method like `EditorScreen.OnExiting()` and move this there? That way at least the bodge would be isolated to the screen.

Reply:
I've applied this unilaterally in https://github.com/ppy/osu/pull/26717/commits/807d982a721ec043990d1871f41316763d3b5b15 to necro this.